infrastructure/iac/template/styles.css

```diff
@@ -0,0 +1,108 @@
+:root {
+  --default-font: "SF Pro Text", "Segoe UI", "Ubuntu", Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
Let's move Tahoma to the second last. According to https://plugins.jetbrains.com/docs/intellij/typography.html, the default fonts are good. In the IDE styles, we should actually set the default font to the value of `JBFont.regular()`